### PR TITLE
Add lists endpoints

### LINKED ISSUE #78: API support for users lists && their members?
Hello @dghubble and other maintainers of `go-twitter`!

On Twitter, a useful feature I employ to keep track of specific niches is the Lists functionality.

The specific API call that I want to make via this library is to get the statuses of all list members. I am using this in a bot I am helping make. It posts new tweets from a users timeline at the moment, and I want to filter those posts to make sure they're only users from the list. I found this API doc below that would allow me to skip that filtering mechanism and directly get statuses only from those list members.

https://dev.twitter.com/rest/reference/get/lists/statuses

I was wondering if there is any pre-existing support in this library (did not see anything in the docs), and if there are plans to integrate such functionality?

General list support would be great 👍  and I would love to help implement this.

### PR DESCRIPTION
**Pull request by peppage**
Closes #78

### CODE REVIEW

File: twitter/lists.go [hunk 1]
```diff
@@ -0,0 +1,434 @@
...
+}
+
+// ListsMembersParams are the parameters for ListsService.Members
+type ListsMembersParams struct {
+	ListID          int64  `url:"list_id,omitempty"`
+	Slug            string `url:"slug,omitempty"`
+	OwnerScreenName string `url:"owner_screen_name,omitempty"`
+	OwnerID         int64  `url:"owner_id,omitempty"`
+	Count           int    `url:"count,omitempty"`
+	Cursor          int    `url:"cursor,omitempty"`
```
Comment (maniak89) [inline twitter/lists.go]: Invalid type of "Cursor" - int64 is needed

**Comment by jscoys** [comment]
Hey guys can we have this thing pushed into the main repo and available for everyone please?

**Comment by stevenleeg** [comment]
Would love to see this merged in as well. Thanks for the work @peppage!

**Comment by dghubble** [comment]
Thanks @peppage and @Mushus ! And thanks for the patience.

I've rebased to resolve conflicts and squashed to keep history nice. This is merged.

**Pull request closed without merging**